homework/index.js

```diff
@@ -29,16 +32,62 @@
     });
     return elem;
   }
+  function app(x) {
```

Comment:
`x` could refer to something specific but it's a short hand when you don't want to use the parameter, other wise you may use a different name describing it's content.